db/src/db.rs

```diff
@@ -396,8 +397,8 @@ impl TypedSQLValue for TypedValue {
             &TypedValue::Boolean(x) => (Value::Boolean(x), ValueType::Boolean),
             &TypedValue::Long(x) => (Value::Integer(x), ValueType::Long),
             &TypedValue::Double(x) => (Value::Float(x), ValueType::Double),
-            &TypedValue::String(ref x) => (Value::Text(x.clone()), ValueType::String),
-            &TypedValue::Keyword(ref x) => (Value::NamespacedKeyword(x.clone()), ValueType::Keyword),
+            &TypedValue::String(ref x) => (Value::Text(x.as_ref().clone()), ValueType::String),
```

Comment:
If we make `edn::Value` use `Rc`, we can avoid ballooning the stream — strings would be pseudo-unique.